src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo-link"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              class="nav-link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
```

Comment:
The 'href' attribute is currently set to '#'. Consider replacing it with an actual URL or path to ensure proper navigation.